Replay/Replay/ViewController.swift

```diff
@@ -276,6 +276,33 @@ class ViewController: UIViewController {
         spinner.stopAnimating()
         spinner.removeFromSuperview()
     }
+    
+    func playerItemArrayCollection(forAsset asset: [AVURLAsset]) -> [AVPlayerItem]{
+        var playerItemArray = [AVPlayerItem]()
+        for mediaAsset in asset{
+            let playerItemTemp = AVPlayerItem.init(asset: mediaAsset)
+            playerItemArray.append(playerItemTemp)
+        }
+        return playerItemArray
+    }
+    
+    func isLastItemPlayed(asset: AVURLAsset) -> Bool {
+        let lastURLOfMedia : String = self.urlArray.last!
+        if(asset.url.absoluteString.range(of: lastURLOfMedia) != nil){
+            return true
+        }
+        else{
+            return false
+        }
+    }
+    
+    func handlePlayback(){
```

Comment:
Lets rename this to something like playNextItem(). This could be in an extension for AVPlayerQueue.